test/e2e/storage/volume_metrics.go

```diff
@@ -165,10 +155,26 @@ var _ = SIGDescribe("[Serial] Volume metrics", func() {
 			kubeletmetrics.VolumeStatsInodesUsedKey,
 		}
 
-		for _, key := range volumeStatKeys {
-			kubeletKeyName := fmt.Sprintf("%s_%s", kubeletmetrics.KubeletSubsystem, key)
-			verifyVolumeStatMetric(kubeletKeyName, pvc.Namespace, pvc.Name, kubeMetrics)
-		}
+		waitErr := wait.Poll(30*time.Second, 5*time.Minute, func() (bool, error) {
+			// Grab kubelet metrics from the node the pod was scheduled on
+			kubeMetrics, err := metricsGrabber.GrabFromKubelet(pod.Spec.NodeName)
+			if err != nil {
```

Comment:
Is there a specific err that indicates that metrics is not ready vs any err?

Reply:
There's isn't as far as I can tell. Also to clarify - when this test flakes - this succeeds, we just don't find the metrics we're interested in updated yet